python/cudf/cudf/core/indexed_frame.py

```diff
@@ -1288,6 +1289,93 @@ def _align_to_index(
 
         return result
 
+    @_cudf_nvtx_annotate
+    def _reindex(
+        self,
+        columns,
```

Comment:
Since it's an internal method, I would recommend calling `columns` `column_names`. I've always used `columns` to refer to an array of cudf columns and `column_names` as the names to these columns. Although this might be a bit inconsistent with pandas, I find it less confusing when reading the code.

Reply:
This should be updated